ui/packages/catalog-ui-search/src/main/webapp/component/input/boolean/input-boolean.view.js

```diff
@@ -13,12 +13,21 @@
  *
  **/
 
+import InputBoolean from './input-boolean.presentation'
+import * as React from 'react'
+
 var Marionette = require('marionette')
 var template = require('./input-boolean.hbs')
```

Comment:
✏️ Can remove this line and the template itself

Reply:
Reverted the change so it will now be required.